meson.build

```diff
@@ -4,13 +4,14 @@ project(
     version: '0.4.0', 
     license: 'GNU General Public License v2 (GPLv2)', 
     # license_files: 'LICENSE.txt',
-    meson_version: '>=1.0.0'
+    meson_version: '>=1.0.0',
+    default_options : ['c_std=c11', 'cpp_std=c++11']
```

Comment:
I don't think `c_std` is needed, since the project is only C++

Reply:
You're right, it works fine without `'c_std=c11'`, but crashes without `'cpp_std=c++11'`.